### DISCUSSION TITLE
Climate control

### DISCUSSION THREAD

**Discussion by Solsind** [pypolestar/polestar_api#233] [Ideas]
Is it possible to control the climate from the HA like it is from the App from Polestar. An example could be if I work and the time for me to drive from of to job, it could start up heating or air condition

**Comment by JohnnyTwoFingers** [comment]
I would like to see this addition too.

**Comment by jschlyter** [comment]
The Polestar API does not yet support mutations (state changes, writes). I've reached out to Polestar trying to find a contact for the API, but no luck yet. When/if we are able to get write access, this should be possible - until then not :-(

**Comment by robspace** [comment]
Just bumping this request to see if you had any luck in contacting Polestar? :-)

**Reply by Sandahlen** [reply]
Or maybe now? :)

**Reply by jschlyter** [reply]
Sorry, Polestar do not want to talk. The have implemented a compliance checkbox "data portal" that is of no use to us. Volvo on the other hand, has several useful APIs available. It's not that Polestar don't know how to do this – they simply don't want to.
